RHEL6/Dockerfile

```diff
@@ -8,8 +8,13 @@ WORKDIR /root
 RUN sed -i -e 's/os.path.exists(HOST_CONFIG_DIR)/False/g' /usr/lib64/python2.6/site-packages/rhsm/config.py
 RUN echo "{\"virt.host_type\": \"Not Applicable\", \"virt.is_guest\": \"False\"}" > /etc/rhsm/facts/custom.facts
 
-ADD startup.sh /tmp/
-RUN chmod +x /tmp/startup.sh
+# add and process the external resources
+ADD resources/* /tmp/
+ADD setup_scripts/* /tmp/setup_scripts/
+RUN ls -l /tmp
+WORKDIR /tmp
+RUN chmod +x *.sh setup_scripts/*.sh || true
```

Comment:
What are the cases where chmod fails? Does chmod behave differently on RHEL 6?

Reply:
$ chmod +x /tmp/fooo*.sh
chmod: cannot access '/tmp/fooo*.sh': No such file or directory.

I only address it here, as this one does not installs `hostname` rpm, so the dirs are empty